lib/flutter_periculum.dart

```diff
@@ -4,7 +4,9 @@ import 'dart:convert';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_periculum/models/AffordabilityResponse.dart';
-import 'package:flutter_periculum/models/ExistingStatment.dart';
+import 'package:flutter_periculum/models/CreditScoreResponse.dart';
+import 'package:flutter_periculum/models/Statement.dart';
```

Comment:
I think we should be consistent and call the file and the name of the model StatementResponse.

Reply:
The line in red means it was remove and change with the blue